runtime-builder/build.sh

```diff
@@ -18,24 +18,37 @@
 # Required arguments:
 #   project - GCP project ID for image name.
 #   go_version - Go SDK version to bundle into image.
-#   debian_tag - Debian tag for application base image.
-
-usage() { echo "Usage: $0 <project> <go_version> <debian_tag>"; exit 1; }
 
 set -e
 
-if [ "$#" -ne 3 ]; then
+usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
+
+debian_digest ()
+{
+  local digest="$(gcloud beta container images describe gcr.io/google_appengine/debian8:latest | \
+    grep '^Image:' | cut -d'@' -f 2)"
+  local prefix="$(echo ${digest} | cut -d':' -f 1)"
+  local len="$(echo ${digest} | cut -d':' -f 2 | wc -c)"
+  if [ "${prefix}" != "sha256" -o "${len}" -ne 65 ]; then
```

Comment:
If we know that $digest includes prefix sha256, then just include the prefix when checking for length, i think that'll be 72.  Do add comment on why 72.

Also, just noticed this.  I think use of [[ ... ]] is preferred over  [ ... ].  Please update use of this below as well.